nexus/src/external_api/console_api.rs

```diff
@@ -354,6 +386,7 @@ pub struct LoginPathParam {
 pub async fn login_local_begin(
     rqctx: RequestContext<Arc<ServerContext>>,
     _path_params: Path<LoginPathParam>,
+    _query_params: Query<LoginUrlQuery>,
```

Comment:
Why is this here?  I see the endpoint is unpublished so it's not to appear in the spec, and it doesn't seem used.

Reply:
The idea is that the redirect_uri is still being validated as a relative path server-side. There’s a new test exercising this I believe.